users/views.py

```diff
@@ -11,3 +14,23 @@ def login(request):
 
 def logout(request):
     return render(request, 'users/logout.html', {'title': 'logout'})
+
+
+@login_required
+def edit_profile(request):
+    if request.method == 'POST':
+        user_form = UserUpdateForm(request.POST, instance=request.user)
+        profile_form = ProfileUpdateForm(request.POST, request.FILES, instance=request.user.profile)
+        if user_form.is_valid() and profile_form.is_valid():
+            user_form.save()
+            profile_form.save()
+            return redirect('users-profile')
+    else:
+        user_form = UserUpdateForm(instance=request.user)
+        profile_form = ProfileUpdateForm(instance=request.user.profile)
+
+    context = {
+        'user_form': user_form,
+        'profile_form': profile_form
+    }
+    return render(request, 'users/edit_profile.html', context)
```

Comment:
Please use [hyphens](https://pyphilly.org/hyphen-and-underscores-when-use-each-django-and-python/) instead of underscore in url
```suggestion
    return render(request, 'users/edit-profile.html', context)
```